mm2src/coins/utxo/utxo_block_header_storage/mod.rs

```diff
@@ -1,10 +1,21 @@
 #[cfg(target_arch = "wasm32")]
-use crate::utxo::utxo_indexedb_block_header_storage::IndexedDBBlockHeadersStorage;
+pub mod indexedb_block_header_storage;
```

Comment:
`indexedb_block_header_storage` and `sql_block_header_storage` can be private modules